### PR TITLE
Proposals API

### PR DESCRIPTION
**Pull request by dthiessen-va**
[Proposals External APIs](https://vendasta.jira.com/wiki/spaces/API/pages/1763411335/Proposals)
Authors: @dthiessen-va @r-LaForge @Wynston 

@vendasta/external-apis

[x] Review the [pre release checklist](https://vendasta.jira.com/wiki/spaces/API/pages/1651769533/Pre+Release+Checks)

### CODE REVIEW

**Review by richard-rance — COMMENTED** [review]
I fixed up a bunch of small stuff while reviewing. There are still a few things that I have questions on. Overall it's looking pretty good.

File: openapi/platform/platform.yaml [hunk 1]
```diff
@@ -5688,419 +6508,651 @@ components:
...
+                - Global
+              default: Organization
+              readOnly: true
+            pdfUrl:
+              type: string
+              readOnly: true
+            previewImageUrl:
+              type: string
+            title:
+              type: string
```
[the remark this answers is not in the document]
Comment (Wynston) [reply openapi/platform/platform.yaml]: Yup!

File: openapi/platform/platform.yaml [hunk 2]
```diff
@@ -3801,6 +3801,548 @@ paths:
...
+      x-lifecycle:
+        status: trustedTester
+    post:
+      security:
+        - OAuth2:
+            - sales.proposals
+      x-lifecycle:
+        status: trustedTester
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
```
[the remark this answers is not in the document]
Comment (richard-rance) [reply openapi/platform/platform.yaml]: You missed updating the status in the x-lifecycle object.

**Review by richard-rance — APPROVED** [review]

File: docs/Overview/Changelog.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 The platform is continuously evolving. This page lists the significant changes when they are announced. For more info on the statuses and release process see [Versioning](./Versioning.md)
 
+## 2022-08-08
+
+### Proposal Builder
+Create, update, get, and list operations added to Platform APIs along with description of [Proposal](../../openapi/platform/platform.yaml)
```
[the remark this answers is not in the document]
Comment (r-LaForge) [reply docs/Overview/Changelog.md]: I honestly had no idea you could do that

[the remark this answers is not in the document]
Comment (richard-rance) [reply docs/Overview/Changelog.md]: Yeah. The link syntax takes a whole page to explain https://meta.stoplight.io/docs/platform/hosted-docs/stoplight-urls#link-to-api-elements

File: openapi/platform/platform.yaml [hunk 4]
```diff
@@ -4338,36 +4914,7 @@ components:
             lineItems:
               type: array
               items:
-                type: object
-                properties:
-                  sku:
-                    type: string
-                    description: 'The unique identifier of a product, product edition or package. It can be obtained by following [this guide](https://docs.apigateway.co/docs/openapi-specs/docs/Guides/Sell/FindSKU.md).'
```
Comment (richard-rance) [inline openapi/platform/platform.yaml:4345]: I missed it before merging but this refactor resulted in changing the field name from `sku` to `id` for the existing salesOrders resource.

Comment (Wynston) [reply openapi/platform/platform.yaml:4345]: Do we want to revert that?

Comment (richard-rance) [reply openapi/platform/platform.yaml:4345]: Just make another PR fixing it. It is only like 2 lines out of this whole PR so not worth reverting it.

Comment (Wynston) [reply openapi/platform/platform.yaml:4345]: Oh okay I also got the fix in this [draft PR ](https://github.com/vendasta/api-gateway-docs/pull/105)as well. Im doing a pass on the proposal builder docs

**Pull request merged**